vllm/config.py

```diff
@@ -744,6 +747,7 @@ class LoadFormat(str, enum.Enum):
     SHARDED_STATE = "sharded_state"
     GGUF = "gguf"
     BITSANDBYTES = "bitsandbytes"
+    CONSOLIDATED = "consolidated"
```

Comment:
CONSOLIDATED is a bit broad as a name no ? Especially since there is SHARDED_STATE above, I think it could lead to confusion.

Thoughts on LoadFormat.MISTRAL ? This makes it clear that the intent is to have us support / maintain the integration of our models into vLLM.

Reply:
Yes makes sense!

Reply:
Changed to "mistral" now - keen to hear what @simon-mo @mgoin think :-)

Reply:
+1 on calling it "MISTRAL" format

Reply:
I agree, it's better to be explicit if possible